frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx

```diff
@@ -109,9 +107,9 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
   }, [cpu, dispatch, memory, nodesCount]);
 
   React.useEffect(() => {
-    const isFlexibleScaling: boolean = scNodeNamesLength && zonesCount < 3;
+    const isFlexibleScaling: boolean = !!(nodesCount >= MINIMUM_NODES && zonesCount < 3);
```

Comment:
Sorry I missed one more thing.

if arbiter is enabled the nodesCount is 3 and zones is 2, flexible sizing will get in place which should not happen. we need to check for if arbiter is enabled then flexible should not be enabled.

Also if you can move this logic to a single place so we don't need to change both always.

Reply:
Okay arbiter is not for internal mode, so we can instead guard on arbiter itself.
We should not mix two diff logics. flexible scaling should stay intact.
I will make that change.

Reply:
for flexible scaling, we have single logic for internal & internal-attached.
attachedDevices
`!!(nodesCount >= MINIMUM_NODES && zonesCount < 3)` 
internal
`!!(nodesCount >= MINIMUM_NODES && zonesCount < 3)`

we can place them somewhere in utils so we don't miss out other if there is a change in one.

Reply:
Okay I guess the ask was for arbiter.

Reply:
Thanks, @afreen23 looks much better now. :)